internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -50,19 +50,25 @@ func (nviu NodeVersionInfoUpdate) HasMajorOrMinorUpdate() bool {
 			nviu.Update.APIServerVersion.Minor() > nviu.Current.APIServerVersion.Minor() {
 			return true
 		}
+		return false
```

Comment:
This should not be here, we want to fall through to checking kubelet and cri-o still.

On control-planes we first check the control plane components, then, if it doesn't have updates on those components, we check the kubelet and cri-o as we do on the rest of the nodes.

Reply:
Thanks,@ereslibre , I got more clear on the checking logic